src/DDTrace/Integrations/Guzzle/V5/GuzzleIntegration.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace DDTrace\Integrations\Guzzle\V5;
+
+use DDTrace\Tags;
+use DDTrace\Types;
+use OpenTracing\Span;
+use DDTrace\Integrations\Integration;
+use GuzzleHttp\Message\ResponseInterface;
+
+class GuzzleIntegration extends Integration
+{
+    const CLASS_NAME = 'GuzzleHttp\Client';
+
+    protected static function loadIntegration()
+    {
+        self::traceMethod('send', function (Span $span, array $args) {
+            $span->setTag('http.method', $args[0]->getMethod());
+            $span->setTag('http.url', $args[0]->getUrl());
```

Comment:
This I forgot to mention: you can use `DDTrace\Http\Urls::sanitize()` to sanitize the url before putting it in the tag.